examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -66,55 +67,60 @@ export class TaskList extends DataObject implements ITaskList {
      * collection pattern -- see the contact-collection example for more details on this pattern.
      */
     private readonly tasks = new Map<string, Task>();
+    private _savedData: SharedMap | undefined;
+    private _draftData: SharedMap | undefined;
+
+    private get savedData(): SharedMap {
+        if (this._savedData === undefined) {
+            throw new Error("The savedData SharedMap was not initialized correctly");
```

Comment:
Nit: for these getter errors, since we don't know the timing in which other logic in this type may call them, it might be better to just say something like "The savedData SharedMap has not yet been initialized". In theory we could call this before natural initialization takes place, in which case the issue is on the caller's side.